setup/www/resources/config/nodejs.org

```diff
@@ -45,6 +45,8 @@ server {
     }
 
     location / {
+        try_files $uri $uri/ @english_fallback;
```

Comment:
Ahhh, that looks smooth. That probably means we could remove the `return 404` line in `@english_fallback` as well?